libs/designsystem/checkbox/src/checkbox.component.scss

```
@@ -125,27 +122,51 @@ $spacing-to-label: map.get(utils.$checkbox-radio-spacing, 'to-label');
   &.has-label {
     display: block;
 
-    @each $size, $height in utils.$checkbox-radio-sizes {
+    @each $size, $properties in utils.$checkbox-sizes {
+      :host-context(.checkbox-#{$size}),
       &.#{$size} {
         ion-checkbox::part(label) {
-          $vertical-padding: get-vertical-padding($height);
+          $vertical-padding: map.get($properties, vertical-padding);
 
           padding-top: $vertical-padding;
           padding-bottom: $vertical-padding;
```

Consider some light boyscouting 😇 
```suggestion
          padding-block: $vertical-padding;
```